tests/test_csc.py

```diff
@@ -266,51 +267,112 @@ async def test_home(self):
+            If angle, min_angle, or max_angle not in range [0, 360)
+            If angle not in the specified range.
+        """
+        for argname in ("angle", "min_angle", "max_angle"):
+            argvalue = locals()[argname]
+            if not 0 <= argvalue < 360:
+                raise AssertionError(
+                    f"Argument {argname} = {argvalue} not in range [0, 360)"
+                )
+        if min_angle > max_angle:
+            if not (min_angle <= angle < 360 or 0 <= angle < max_angle):
+                raise AssertionError(
+                    f"angle {angle} not in range [{min_angle}, 360) or [0, {max_angle}]"
+                )
+        else:
+            if not (min_angle <= angle < max_angle):
+                raise AssertionError(
+                    f"angle {angle} not in range [{min_angle}, {max_angle}]"
+                )
+
+    async def test_move_azimuth(self):
         async with self.make_csc(
             initial_state=salobj.State.ENABLED, config_dir=None, simulation_mode=1
         ):
             await self.check_initial_az_events()
 
-            desired_azimuth = 354
-            await self.remote.cmd_moveAzimuth.set_start(
-                azimuth=desired_azimuth, timeout=STD_TIMEOUT
-            )
+            # Try several angles, including some not in the range [0, 360);
+            # CW direction is towards larger azimuth
+            isFirst = True
+            for desired_azimuth, desired_moving_state, min_angle, max_angle in (
+                (325.1, AzimuthState.MOVINGCCW, 325.1, 0),
+                (-1.2, AzimuthState.MOVINGCW, 325.1, 360 - 1.2),
+                (390.3, AzimuthState.MOVINGCW, 360 - 1.2, 390.3 - 360),
+            ):
+                wrapped_desired_azimuth = salobj.angle_wrap_nonnegative(
+                    desired_azimuth
+                ).deg
+                await self.remote.cmd_moveAzimuth.set_start(
+                    azimuth=desired_azimuth, timeout=STD_TIMEOUT
+                )
+                if not isFirst:
+                    await self.assert_next_sample(
+                        topic=self.remote.evt_azimuthInPosition, inPosition=False,
+                    )
 
-            # wait for the move to begin and check status
-            az_cmd_state = await self.assert_next_sample(
-                topic=self.remote.evt_azimuthCommandedState,
-                commandedState=AzimuthCommandedState.GOTOPOSITION,
-            )
-            self.assertAlmostEqual(az_cmd_state.azimuth, desired_azimuth)
-            await self.assert_next_sample(
-                topic=self.remote.evt_azimuthState,
-                state=AzimuthState.MOVINGCCW,
-                homing=False,
-            )
-            position = self.remote.tel_position.get()
-            self.assertGreater(position.azimuthPosition, desired_azimuth)
-            self.assertLess(position.azimuthPosition, 360)
+                # wait for the move to begin and check status
```

Comment:
All your comments start with upper case letters except this one.

Reply:
OK. It's quite a mix in the file. I'll make them all start with uppercase and end with a period (or question mark).